SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -201,8 +202,17 @@ public SingularityRequestParent bounce(@ApiParam("The request ID to bounce") @Pa
 
     requestManager.bounce(requestWithState.getRequest(), System.currentTimeMillis(), JavaUtils.getUserEmail(user), message);
 
+    SingularityBounceRequest defaultBounceRequest = bounceRequest.or(SingularityBounceRequest.defaultRequest());
```

Comment:
last nitpick on this, since this can be the user-provided data or the default, `default` doesn't make sense as a variable name. Maybe something like `effectiveBounceRequest` (or something similar??)